lecli/cli.py

```diff
@@ -326,9 +327,93 @@ def deleteuser(userkey):
 @cli.command()
 def getowner():
     """Get account owner details"""
-
     user_api.get_owner()
 
 
+@cli.command()
+@click.option('-n', '--name', type=click.STRING, default=None,
+              help='Name of new log')
+@click.option('-f', '--filename', type=click.STRING, default=None,
+              help='File containing json representation of log object')
+def createlog(name, filename):
+    """Create a log with the provided name and details.
+    Will use JSON file first if both name and file are provided"""
+
+    if filename is not None:
+        with open(filename) as json_data:
+            # Open file and load as JSON
+            try:
+                params = json.load(json_data)
+                log_api.create_log(None, params)
+            except ValueError as error:
+                print error
+                exit(1)
+    elif name is not None:
+        log_api.create_log(name, None)
+    else:
+        click.echo('Example usage: lecli createlog -n \'new_log_name\'')
+        click.echo('Example usage: lecli createlog -f \'path_to_file.json\'')
+
+
+
+@cli.command()
+@click.argument('logid', type=click.STRING, default=None)
+def deletelog(logid):
+    """Delete a log with the provided id"""
+    log_api.delete_log(logid)
+
+
+@cli.command()
+def getlogs():
+    """Get all logs"""
+    log_api.get_logs()
+
+
+@cli.command()
+@click.argument('log_id', type=click.STRING, default=None)
+def getlog(log_id):
+    """Get a log with the provided id"""
+    log_api.get_log(log_id)
+
+@cli.command()
+@click.argument('log_id', type=click.STRING)
+@click.argument('name', type=click.STRING)
+def renamelog(log_id, name):
+    """Update a log with the provided details"""
+    log_api.rename_log(log_id, name)
+
+
+@cli.command()
+@click.argument('log_id', type=click.STRING)
+@click.argument('filename', type=click.STRING)
+def updatelog(log_id, filename):
+    """Update a log with the details provided"""
+    with open(filename) as json_data:
+        # Open file and load as JSON
+        try:
+            params = json.load(json_data)
+            log_api.update_log(log_id, params)
+        except ValueError as error:
+            print error
```

Comment:
maybe provide a better example and it should probably be printed to sys.stderr instead of stdout